README.md

```diff
@@ -36,6 +36,14 @@ This SDK lets you:
 
 Additionally, the SDK includes classes for work with geospatial tiling schemes that are used by most OLP catalog layers.
 
+## Backward Compatibility
+
+We try to develop and maintain our API in a way that preserves its compatibility with the existing applications. Changes in our API are greatly influenced by the Data Service development. Data Service introduces breaking changes 6 months in advance. Therefore, you may need to migrate to a new version of our SDK every half a year.
+
+For more information on Data Service, see its [Developer Guide](https://developer.here.com/olp/documentation/data-api/data_dev_guide/index.html) and [API Reference](https://developer.here.com/olp/documentation/data-api/api-reference.html).
+
+When new API is introduced, we do not delete the old one straight away. Our standard deprecation time is 6 months. It gives you time to switch to new code. All of the deprecated methods, functions, and parameters are documented in our [API Reference](https://developer.here.com/olp/documentation/sdk-cpp/api_reference/index.html) and [changelog](CHANGELOG.md).
```

Comment:
Maybe we should make clearer that this is about the API changes in our SDK vs the changes in the Data Service?

Reply:
Modified